src/mui/delete/Delete.js

```diff
@@ -56,13 +56,20 @@ class Delete extends Component {
             .join('/');
     }
 
+    defaultRedirectRoute() {
+        return 'list';
+    }
+
     handleSubmit(event) {
         event.preventDefault();
         this.props.crudDelete(
             this.props.resource,
             this.props.id,
             this.props.data,
-            this.getBasePath()
+            this.getBasePath(),
+            this.props.redirect
+                ? this.props.redirect
+                : this.defaultRedirectRoute()
```

Comment:
useless call: can you put `list` directly here or in a const if you prefer

Reply:
well i copied it from `Edit.js` .)

Reply:
but i guess i can just remove the condition directly as `crudDeleteAction` has default 'list'

Reply:
Ah right, well that looks strange to me but anyway, ignore this request